web/components/contract/contract-tabs.tsx

```diff
@@ -94,6 +103,8 @@ export function ContractTabs(props: {
     (totalPositions > 0 ? `${shortFormatNumber(totalPositions)} ` : '') +
     maybePluralize('Holder', totalPositions)
 
+  const unfilledBets = useUnfilledBets(liveContract.id) ?? []
```

Comment:
It'd be best to only load these bets if someone has toggled the order book, not on every page load.